apps/arena/lib/arena/game/player.ex

```diff
@@ -168,6 +169,20 @@ defmodule Arena.Game.Player do
         game_state
 
       skill ->
+        Process.send_after(
+          self(),
+          {:block_actions, player.id},
+          skill.execution_duration_ms
+        )
```

Comment:
Why do you add this :block_actions message?

Reply:
I think this was a conflict with the dynamic leap execution duration pr  since this call was repeated 😂 i deleted this and moved the movement call to use the same duration as the action